frontend/src/components/widgets/Button/Button.tsx

```diff
@@ -27,6 +29,29 @@ export interface ButtonProps {
   width: number
 }
 
+interface ButtonTooltipProps {
+  children: ReactElement
+  help?: string
+}
+
+function ButtonTooltip(props: ButtonTooltipProps): ReactElement {
+  const { children, help } = props
```

Comment:
nit: (not sure if this is being done elsewhere, but if not done anywhere else then we probably shouldn't here for consistency) but `prop`s can be destructured in the function signature

```javascript
function ButtonTooltip({children, help}: ButtonTooltipProps): ReactElement {
    // ...
}
```